ui/src/app/components/inputs.tsx

```diff
@@ -148,7 +148,6 @@ interface TextAreaWithLengthValidationMessageProps {
   id: string;
   initialText: string;
   maxCharacters: number;
-  tooLongWarningCharacters: number;
```

Comment:
This prop was required via this interface but was not used by the component that implements the interface. I removed the prop from the interface and all instances of TextAreaWithLengthValidationMessage. If it was there for a reason, maybe we could change it to optional.

Reply:
removal looks right to me, thanks